frontend/src/stateless/index.ts

```diff
@@ -0,0 +1,344 @@
+import { indexedDB as indexedDBtest } from 'fake-indexeddb';
```

Comment:
This testing module shouldn't really be imported here.

Maybe it's not even necessary? If the tests can run without it, it would be good to avoid adding the dependency.

Reply:
This code is for testing, and it should only be imported by the test code.

Reply:
Thanks for the suggestion, moving this to a test file and injecting it here.